components/user-mgt/org.wso2.carbon.user.mgt.ui/src/main/resources/web/user/user-mgt.jsp

```diff
@@ -241,19 +242,36 @@
 
                 countableUserStores = countClient.getCountableUserStores();
 
-                if (UserAdminUIConstants.SELECT.equalsIgnoreCase(countClaimUri)) {    //this is user name based search
-                    if (selectedCountDomain.equalsIgnoreCase(UserAdminUIConstants.ALL_DOMAINS)) {
-                        userCount = countClient.countUsers(countFilter);
-                    } else {
-                        userCount.put(selectedCountDomain, String.valueOf(countClient.countUsersInDomain(countFilter, selectedCountDomain)));
-                    }
-                } else {                //this is a claim based search
-                    if (selectedCountDomain.equalsIgnoreCase(UserAdminUIConstants.ALL_DOMAINS)) {
-                        userCount = countClient.countByClaim(countClaimUri, countFilter);
-                    } else {
-                        userCount.put(selectedCountDomain, String.valueOf(countClient.countByClaimInDomain(countClaimUri,
-                                countFilter, selectedCountDomain)));
+                String countingUserDomain = null;
+                try {
+                    if (UserAdminUIConstants.SELECT.equalsIgnoreCase(countClaimUri)) {    //this is user name based search
+                        if (selectedCountDomain.equalsIgnoreCase(UserAdminUIConstants.ALL_DOMAINS)) {
+                            countingUserDomain = UserAdminUIConstants.ALL_DOMAINS;
+                            userCount = countClient.countUsers(countFilter);
+                            errorGettingUserCount = false;
+                        } else {
+                            countingUserDomain = selectedCountDomain;
+                            userCount.put(selectedCountDomain, String.valueOf(countClient.countUsersInDomain
+                                (countFilter, selectedCountDomain)));
+                            errorGettingUserCount = false;
+                        }
+                    } else {                //this is a claim based search
+                        if (selectedCountDomain.equalsIgnoreCase(UserAdminUIConstants.ALL_DOMAINS)) {
+                            countingUserDomain = UserAdminUIConstants.ALL_DOMAINS;
+                            userCount = countClient.countByClaim(countClaimUri, countFilter);
+                            errorGettingUserCount = false;
+                        } else {
+                            countingUserDomain = selectedCountDomain;
+                            userCount.put(selectedCountDomain, String.valueOf(countClient.countByClaimInDomain
+                                (countClaimUri, countFilter, selectedCountDomain)));
+                            errorGettingUserCount = false;
+
+                        }
                     }
+                } catch (Exception e) {
```

Comment:
Any reason to catch 'Exception' instead of specific exception(s)? If a method throws that in the same type it's fine.